src/components/AccountDetails/TransactionSummary.tsx

```diff
@@ -87,7 +88,11 @@ function SubmitProposalTransactionSummary() {
 function ApprovalSummary({ info }: { info: ApproveTransactionInfo }) {
   const token = useToken(info.tokenAddress)
 
-  return <Trans>Approve {token?.symbol}</Trans>
+  return BigNumber.from(info.amount)?.eq(0) ? (
+    <Trans>Revoke {token?.symbol}</Trans>
```

Comment:
so this is actually happening because we can't currently know what the user types into the metamask box. 

it works in the USDT case because we are explicitly requesting an allowance of 0, but in your case we're requesting MAX_ALLOWANCE and metamask just allows you to override that to zero.

it's a bigger problem that we need to figure out, because the permit2 flow actually fails if you allow a too-small amount

Reply:
i'd like to resolve this in a separate PR if that works for you